### ISSUE TITLE
Should lint on not-locally used private data

### ISSUE THREAD

**Issue by radbasa** [Appsilon/box.linters#34]
## box.linters version

0.0.0.9006

## Source code to lint

```R
#' @export
public_function <- function() {
  \"exported function\"
}

private_data <- c(1, 3)
```

## Lint command

```R
lintr::lint(code, linters = lintr::linters_with_defaults(defaults = box.linters::rhino_default_linters))
```

## Expected result

Should lint on unused `private_data`.
